libguard/guard_interface.hpp

```diff
@@ -63,6 +63,14 @@ void clear(const uint32_t recordId);
  */
 void clearAll();
 
+/**
+ * @brief Clearing guard records with invalid recordId
+ *        i.e. recordId set as 0xFFFFFFFF.
+ *
+ * @return NULL
+ */
+void clearInvalidRecords();
```

Comment:
Do you need this support in the guard tool?

Reply:
Need to discuss.